llama_index/agent/types.py

```diff
@@ -170,12 +170,25 @@ class Task(BaseModel):
     )
 
 
-class BaseAgentWorker(ABC):
+class BaseAgentWorker(PromptMixin):
     """Base agent worker."""
 
     class Config:
         arbitrary_types_allowed = True
 
+    def _get_prompts(self) -> PromptDictType:
+        """Get prompts."""
+        # TODO: the ReAct agent does not explicitly specify prompts, would need a
+        # refactor to expose those prompts
```

Comment:
Apologies for likely a dumb question: but do you mind explaining what you mean here?

Reply:
oops i'm just gonna remove